subsystems/win32/csrsrv/init.c

```diff
@@ -153,7 +159,7 @@ CsrSetProcessSecurity(VOID)
     /* Free the memory and return */
 Quickie:
     if (ProcSd) RtlFreeHeap(CsrHeap, 0, ProcSd);
-    RtlFreeHeap(CsrHeap, 0, TokenInfo);
+    if (TokenInfo) RtlFreeHeap(CsrHeap, 0, TokenInfo);
```

Comment:
```suggestion
    if (TokenInfo)
        RtlFreeHeap(CsrHeap, 0, TokenInfo);
```

Reply:
Let's not modify in an non-consistent manner the formatting, please.